### DISCUSSION TITLE
Synthesize TTW Core Team meeting questions about Governance

### DISCUSSION THREAD

**Discussion by AlexandraAAJ** [the-turing-way/the-turing-way#3204] [Governance]
## Structuring questions under subsections

Following the feedback received from the meeting on 09 June, we have synthesize all the questions and comments in 9 groups:

- Book
- Book dash
- Communications
- Community participation guideline
- Funding 
- Github
- Governance / Core team
- Project wide
- TPS

Each group will answer the following question:
**"is the answer in the book"** with options YES or NO. 

- If the answer is YES: a link will be shared 
- If the answer is NO we will add a line with next steps

### Book

| Question | Is the answer in the book | Link / Comment |
| -------------- |-----| -----| 
|  What content ends up in the book | - [ ] Yes  | [Ways of working section](https://github.com/alan-turing-institute/the-turing-way/blob/main/ways_of_working.md) - Project members - review, assign and approve content. Here are also ways to contribute [Community members, contributors and co-authors](https://the-turing-way.netlify.app/community-handbook/acknowledgement/acknowledgement-members.html#community-members-contributors-and-co-authors) 
Who designs fun new things like a new landing page?| - [ ] No | Design suggestions have happened on an adhoc basis, with more experimental ideas usually being acted on at Book Dash
Who decides if an edit should be a new chapter or just an edit? | - [ ] Yes | [Ways of working section](https://github.com/alan-turing-institute/the-turing-way/blob/main/ways_of_working.md) - Project members - could be more clear but it is there
Who to consult if you want to revise or restructure a chapter? | - [ ] Yes  |[Ways of working section](https://github.com/alan-turing-institute/the-turing-way/blob/main/ways_of_working.md) - Project members - could be more clear but it is there
How long can sections/chapters etc be? |- [ ] No | The [Chapter Template Folder](https://github.com/alan-turing-institute/the-turing-way/tree/main/book/templates/chapter-template) contains lots of relevant info but not exactly the lenght of the chapters/sections
How to represent whose opinion's is reflected in a subchapter/chapter (not even resolving, just showing)? | - [ ] No |(No idea) Other team member's answer: Enhancement Proposals? [democratic mediums](https://medlabboulder.gitlab.io/democraticmediums/). It should definitely be a simpler process than JEPs :sweat_smile:  Django
What is and is **not** appropriate content for the book? +1+1 | - [ ] No |Project members review and approve commits,issues and PR. They can identify NOT appropriate content for the book or raise it for discussion - Make it more clear? [Ways of working section](https://github.com/alan-turing-institute/the-turing-way/blob/main/ways_of_working.md) 
Who decides if content is out of scope for TTW? | - [ ] No |The community monitors and adapts the content of the book and the project members review, guide and approve changes
How do we avoid becoming overly broad? | - [ ] No  | 
How to manage citation/giving credit to the pulling together of ideas | - [ ] Yes |   [Citing The Turing Way section](https://the-turing-way.netlify.app/index.html#citing-the-turing-way)

Questions that are repeated in some way in this section:
- What content is appropriate for TTW 
- TTW will never be finished - but how do we define when things are out of scope and should not be added to the book? is there a point where we have all the themes or guides that are necessary and we keep building on those?

### Book Dash

| Question | Is the answer in the book | Link / Comment |
| -------------- |-----| -----| 
Who comes to the bookdash? | - [ ] Yes | [ Who should apply to join the Book Dash](https://the-turing-way.netlify.app/community-handbook/bookdash.html)
Who sets the timelines for bookdash? | - [ ] No
Who designs the structure of the bookdash? | - [ ] No
What projects get accepted to bookdash? What is the overall objective of the Book Dash? | - [ ] Yes | [Book dash Objectives](https://the-turing-way.netlify.app/community-handbook/bookdash.html)
Who approves images (scriberia drafts)? | - [ ] No
book dash acceptance +1+1 (accountability to the line manager) | - [ ] Yes | [Selection of book dash participants](https://the-turing-way.netlify.app/community-handbook/bookdash/bookdash-selection.html#ch-bookdash-preparation)
  
### Communications

| Question | Is the answer in the book | Link / Comment
| -------------- |-----| -----| 
Who gives talks about the turing way | - [ ] | 
Who applies & who is given the opportunity to talk at conferences | - [ ] |
Who represents TTW at conferences/events in an official capacity | - [ ] |
What events/training etc we run | - [ ] |
How long is too long to leave a post unasnwered in TTW Slack | - [ ] |
What talks and event opportunities are available | - [ ] |
slack is a lot - important messages can be lost quickly (many people are in multiple workspaces) | - [ ] No | We have a [slack policy](https://hackmd.io/@turingway/ttw-slack-intro), which will be added to the book

### Community Participation guideline

| Question | Is the answer in the book | Link / Comment |
| -------------- |-----| -----| 
How are conflicts between members resolved? | No | 
Who writes the CoC? | Yes | It is at the end of the section, however should the [CoC Committee](https://github.com/alan-turing-institute/the-turing-way/blob/main/CODE_OF_CONDUCT.md) be updated? Points of contact should be updated & training done with core delivery staff
Who makes decisiosn on potential breaches of COC? |  Yes | There's a [CoC Committee](https://github.com/alan-turing-institute/the-turing-way/blob/main/CODE_OF_CONDUCT.md)
Who enforces the CoC? | - [ ] Yes| [CoC Committee](https://github.com/alan-turing-institute/the-turing-way/blob/main/CODE_OF_CONDUCT.md)
Who can bring a topic to a collaboration cafe? | - [ ] Yes | [Collab Cafe Topic proposals](https://the-turing-way.netlify.app/community-handbook/coworking/coworking-collabcafe.html)
Who can bring a guest to a collaboration cafe? | - [ ] No 
Who joins coworking on Monday? | - [ ] Yes | While Kirstie has sent this by email, it is not in the [Weekly Coworking calls section](https://the-turing-way.netlify.app/community-handbook/coworking/coworking-weekly.html#ch-coworking-weekly)

### Funding

| Question | Is the answer in the book | Link / Comment |
| -------------- |-----| -----| 
How to get funding to go to a conference | - No | We don't have a policy in place
Who decides to apply for funding for The Turing Way? | - No
Which events does TTW sponsor? | - [ ] No| TTW has no funding available to sponsor 
How do I get further information of the funding (past, present, future) of the TTW? | - [ ]
How funds are spent? | - [ ] No | This should be connected to the Turing Institute participation and how they pay core staff salaries
Who can apply for funding on behalf of the project? | - [ ] No 

Repeated questions/ reflections:
* funding allocation 
    * TTW team will take this on

### GitHub 

Comment from Alex to Anne (and all), some of the questions here are very similar to "the book" ones, what about merging these 2 sections into "Github content"?

| Question | Is the answer in the book | Link / Comment |
| -------------- |-----| -----| 
What tools do we use? | - [ ]   | ?
How do we utilise GitHub?| - [ ] | ?
Who should review a PR* | - [ ] Yes | Project members - review, assign and approve PR. The name of the section could be changed for better understading? [Ways of working section](https://github.com/alan-turing-institute/the-turing-way/blob/main/ways_of_working.md)
Who do I _nominate_ to review a PR | -[ ] Yes | Project members - review, assign and approve PR. The name of the section could be changed for better understading? [Ways of working section](https://github.com/alan-turing-institute/the-turing-way/blob/main/ways_of_working.md)
When a piece of work is "done" enough to submit a PR +1 | - [ ] No | This is not explicitly mentioned. The contributor can decide it and the reviewer will approve or add comments. 
Who should fix a broken link if it is urgent (eg, the home page doesn't load)? | - [ ] No   |    There's a [get in touch](https://github.com/alan-turing-institute/the-turing-way#get-in-touch) section.. We could specify urgent issues. 
Who should fix a broken link if it is not urgent?| - [ ] No | The [Get in touch](https://github.com/alan-turing-institute/the-turing-way#get-in-touch) section could include this type of issues
Who resolves differences of opinion between the author of a PR and the reviewer? | - [ ] No | This sounds like something that *officially* should be in the remit of the CM as a mediator, but may be very context dependent
Who merges a PR? | - [ ] Yes | Project members - review, assign and approve PR. The name of the section could be changed for better understading? [Ways of working section](https://github.com/alan-turing-institute/the-turing-way/blob/main/ways_of_working.md)    

### Governance/Core team

| Question | Is the answer in the book | Link / Comment |
| -------------- |-----| -----| 
What roles people have | No | *Contributor roadmap could help*
Who is in charge +1+1 | No | *Pathway - Roadmap to be developed*
Who becomes core  +1 | No | *Pathway - Roadmap to be developed*
What it means to be on the core team +1 | No | *Pathway - Roadmap to be developed*
What makes you a core member? How much time you should give away as a core member? | - [ ] Almost Yes | In the book we call them [Project Members](https://github.com/alan-turing-institute/the-turing-way/blob/main/ways_of_working.md), we have the list and how much time they should be giving
Who is in the working groups | - [ ] No | *Pathway - Roadmap to be developed* | 
Who decides when working groups meet? | - [ ] No | Working groups are not mentioned in the book, but groups have decided when to meet as needed
Who is a working group accountable to? | - [ ] No | *Pathway - Roadmap to be developed* -> Suggested: chairs, co-chairs, core? 
Who creates a working group? what is expected from working group?+1 | - [ ] No | So far, groups have been created with help from CM, identifying already existing workstreams in the project. A template needs to be developed for working group creation
How autonomous is a working group | - [ ] No | 
Who governs the working group, can we just join a different one that aligns with our work better | - [ ] No
When to change a policy, process, tool (e.g. if we think something new is better) | - No | *Contributor roadmap could help*
How to participate in decision making / what is appropriate to be fully open and democratic and what isn't - many decisions are very transparent. But difficult to know where to find/be in the room for. +1 | - [ ] No | *Pathway - Roadmap to be developed*
How can a regular community member have decision making power? should they? | - [ ] No   
How to agree consensus from the community ? | - [ ] No

Questions/Reflections that are repeated in some way in this section:

* Is anyone in charge? (should anyone be?)+1
* Is there a pathway to decide who's in charge? +1
* Who decides how the volunteers are recruiting to the core team? +1
* How do other working groups operate? Do they need to consult the community? 
* how are people held to account?
* * how to participate in decision making / what is appropriate to be fully open and democratic and what isn't - many decisions are very transparent. But difficult to know where to find/be in the room for. +1
    * Pathway

### Project-wide

| Question | Is the answer in the book | Link / Comment |
| -------------- |-----| -----| 
Who decides things in TTW? | - [ ] No | Work in progress (Governance)
What the direction of the Turing Way is - what is the vision, how are we moving towards it +1 | - [ ] No |Work in progress (Governance)
Relationship between the Turing Institute and The Turing Way +1 million +1+1+1+1 | - [ ]Yes | [In our Communiy section](https://the-turing-way.netlify.app/index.html), it says: "Although The Turing Way receives support and funding from The Alan Turing Institute, the project is designed to be a global collaboration." It could be better explained 
What an external collaboration with ttw could look like | - [ ] No | Very interested to hear how to guide people here, I guess we could use The Practitioners Hub as an example(?)
What are the goals of the project? What is our manifesto? Are we still/just a "reproducible, ethical and collaborative data science"? +1 | - [ ] No | Has been noted a few times in different contexts - work in progress
who do we consult for big changes e.g. Jupyter | - [ ] No | [Get in touch](https://github.com/alan-turing-institute/the-turing-way#about-the-project) could include this or we could add a contributor roadmap for big/simple changes (Anne, I think I've seen something you prepared)
transparency vs not being able to find things | - ? | Should we include a Q&A section in the book
Power of relationships & connections as a means of acquiring information | - [ ] No |The information should be easy to find to avoid structures of power. *Should this be in the onboarding process ?*

Questions/Reflections that are repeated in this section:
* acessibility vs transparency are different things
* don't want lists of lists to find things
* user experience of community contribution
* difficult to manage hardcore transparency with importance of building relationships
* what's the difference between being 'community-led' and being 'community-inspired' or 'community-informed'
* turing way core delivery staff will draft a version of the document to share more widely with the core team
    * role of core team is to hold the core delivery staff accountable to reporting out and listening to feedback
* Meta point: There is a difference between not being transparent and information not being known (hard to find, not promoted, not consulted, _etc._) +1 (this was our b/o room's EXACT point) +1

### TPS

| Question | Is the answer in the book | Link / Comment |
| -------------- |-----| -----| 
How much contribution is expected from TPS team members? | - [ ] Yes

Other comments that are already included in the above groups. 
* turing people in general say please cut down on emails
* most visible issue mentioned above is the relationship between the turing institute and the turing way (especially regarding funding allocation)
    * how can we make this relationship more clear?
* is Kirstie the BDFL? (https://en.wikipedia.org/wiki/Benevolent_dictator_for_life)
    * emergently, KW is, but written into founding documents of TTW that this should not be the case
    * upcoming governance documentation should include information about this
    * design of the turing way should be that it should be able to exist within the turing institute 
    * important to tease apart the relationship with tps & staff members
    * when should a coup happen???? (lol - jk for the reader)
    * role of power dynamics & hierarchies, and teasing out these relationships
    * up to what point does the turing way have a specific vision (that comes from the turing) and the community-led vision (coming from)

**Comment by aleesteele** [comment]
@AlexandraAAJ - is there a reason why this is locked? It's great to see these questions formatted as tables!

**Reply by AlexandraAAJ** [reply]
Not that I remember, I think we wanted to centralised the Governance conversation in this issue (https://github.com/the-turing-way/the-turing-way/discussions/3215) and therefore didn't used this format.
